primeqa/ir/dense/dpr_top/dpr/config.py

```diff
@@ -0,0 +1,195 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class DPRTrainingArguments:
```

Comment:
For now, I don't see any issues but in future we might have use a standard config defining practice across PrimeQA and all utility methods parsing config should be part of that. 


Also we have to define a common mechanism to set something as a `choices` or `options` in metadata for a field.